src/components/App.jsx

```diff
@@ -150,16 +150,15 @@ class App extends React.Component {
   };
 
   onToggleChangeDarkMode = event => {
-    if(event.target.checked){
-      document.body.className = "dark-mode";
+    if (event) {
+      document.body.className = 'dark-mode';
       this.setState({
-        darkMode: true
+        darkMode: true,
       });
-    }
-    else{
-      document.body.className = "";
+    } else {
+      document.body.className = '';
       this.setState({
-        darkMode: false
+        darkMode: false,
```

Comment:
Thanks for getting these formatting changes in 👍  I might need to add an issue to a pre-commit hook for linting